packages/react-dom/src/events/forks/EventListener-www.js

```diff
@@ -16,16 +16,18 @@ export function addEventBubbleListener(
   element: Element,
   eventType: string,
   listener: Function,
+  passive?: boolean,
 ): void {
-  EventListenerWWW.listen(element, eventType, listener);
+  EventListenerWWW.listen(element, eventType, listener, passive);
```

Comment:
This doesn't seem right. The internal EventListener doesn't accept a 4th argument (and Event.listen accepts a forth argument which is a priority and won't work with this).

Reply:
I’ll update the www version